dataplane/packet/packet.c

```diff
@@ -217,3 +218,8 @@ parse_packet(struct packet *packet) {
 
 	return 0;
 }
+
+struct packet *
```

Comment:
This change may have a significant impact on performance, but I hope the LTO must be the thing that makes this change irrelevant.

Reply:
I think the performance degradation will not be significant, so as we do not have any measurements now let merge it. In any case we could manage this in the future.